### ISSUE TITLE
Player teleport

### ISSUE THREAD

**Issue by Helyxst** [jklmao/ClickTpa#4]
Its possible to add an option to the player get teleported to a safe place? 
Players that are flying and accept it, if the players isn't flying they will die.

**Comment by jklmao** [comment]
Yes, thank you for bringing this issue to my attention. I'll look into it for a future update. Thanks!
